spec.bs

```diff
@@ -633,12 +633,16 @@ To <dfn>generate and score bids</dfn> given an [=auction config=] |auctionConfig
         1. Let |biddingScript| be the result of [=fetching resource=] with |ig|'s
           [=interest group/bidding url=], and "text/javascript". If |biddingScript| is an error,
           [=iteration/continue=].
-        1. TODO: Let |generatedBid| be the result of [=evaluating a bidding script=] with
+        1. TODO: Let |generatedBidResult| be the result of [=evaluating a bidding script=] with
            |biddingScript| and « |interestGroup|, |auctionSignals|, |perBuyerSignals|,
            |trustedBiddingSignals|, |browserSignals| ».
-        1. If |generatedBid|'s [=generated bid/bid=] is negative or 0, [=iteration/continue=].
+        1. If |generatedBidResult| is an [=abrupt completion=], [=iteration/continue=].
+        1. Let |generatedBidIdl| be the result of [=converted to an IDL value|converting=] |generatedBidResult| to

```

Comment:
nit `generatedBidIDL` (this is kind of a funny variable name, but I don't have a better suggestion 🤷). Here and elsewhere.

Reply:
Done